src/Couchbase.Lite.Shared/Sync/HTTPLogic.cs

```diff
@@ -208,7 +208,7 @@ private static string GetUserAgent()
 
 			var runtimePlatform = Service.GetInstance<IRuntimePlatform>();
 			var osDescription = runtimePlatform?.OSDescription ?? RuntimeInformation.OSDescription;
-			var hardware = runtimePlatform != null ? $"; {runtimePlatform.HardwareName}" : "";
+			var hardware = runtimePlatform?.HardwareName != null ? $"; {runtimePlatform.HardwareName}" : "";
```

Comment:
total niggle....
would it not be clearer to say:
`var hardware = runtimePlatform?.HardwareName == null ? "" : $"; {runtimePlatform.HardwareName}";`